### PR TITLE
fix: udpates to how carbon works

### PR DESCRIPTION
**Pull request by aubrey-oneal**
Relates #941 

#### Changelog

**Changed**
- Content changes to How Carbon Works based on our updated naming convention.
- Updated ecosystem image (this image is semi-finalized)

#### Testing / reviewing

Request for content review

### CODE REVIEW

**Comment by aubrey-oneal** [comment]
I have read the DCO document and I hereby sign the DCO.

**Review by andreancardona — APPROVED** [review]
Looks good to me! :)

**Comment by yumeforever** [comment]
@aubrey-oneal To me it looks great, I like how you reworded libraries, design kits, and assets. Pretty good!

File: services/web-app/pages/about-carbon/how-carbon-works.mdx [hunk 1]
```diff
@@ -16,294 +16,65 @@ import { section } from './how-carbon-works.module.scss'
...
+As Carbon has evolved, so has our language. Below are a few of the most commonly used terms.
 
-![Marc Johlic headshot](/about-carbon/team/johlic_marc.png)
+**Elements** translate the IBM Design Language into resources for building digital experiences, such
+as color tokens, type tokens, spacing tokens, the grid, and more.
 
-</Profile>
-<Profile name="Craig Moser" title="Vice President of Design and Usability, AI Applications">
+**Guidelines** provide direction on critical topics such as accessibility and content and ensure
+consistent user experiences that meet IBM`s standards.
```
Comment (kingtraceyj) [inline services/web-app/pages/about-carbon/how-carbon-works.mdx:55]:
```suggestion
consistent user experiences that meet IBM's standards.
```

a small apostrophe update!

**Review by kingtraceyj — CHANGES_REQUESTED** [review]
just a small apostrophe update! it's looking really good.

i don't have my dev environment set up currently so i can't look at the image in context. we usually export the image at 2x to make sure the resolution is ok.

**Pull request merged**